pkg/minishift/clusterup/clusterup.go

```diff
@@ -130,7 +135,7 @@ func PostClusterUp(clusterUpConfig *ClusterUpConfig, sshCommander provision.SSHC
 
 // EnsureHostDirectoriesExist ensures that the specified directories exist on the VM and creates them if not.
 func EnsureHostDirectoriesExist(host *host.Host, dirs []string) error {
-	cmd := fmt.Sprintf("sudo mkdir -p %s", strings.Join(dirs, " "))
+	cmd := fmt.Sprintf("sudo install -d -o docker -g docker -m 755 %s", strings.Join(dirs, " "))
```

Comment:
@praveenkumar I am not sure what we are doing here? I am assuming you will again change this for 3.10. 
You might want to change the function description in comments.

Reply:
@LalatenduMohanty  this is not going to change for 3.10, I think we should have done earlier because this command make sure that whatever directory we create should have `docker` user ownership instead root. even oc cluster up also create it same way.